src/js/content.js

```diff
@@ -60,21 +61,32 @@ function openPopup(payload) {
   if (!document.getElementById("moco-bx-popup-root")) {
     const domRoot = document.createElement("div")
     domRoot.setAttribute("id", "moco-bx-popup-root")
-    document.body.appendChild(domRoot)
+    const rootNode = document.querySelector("[aria-modal]") || document.body
+    rootNode.appendChild(domRoot)
   }
 
+  const container = document.getElementById("moco-bx-popup-root")
+
   if (!popupRoot) {
-    const container = document.getElementById("moco-bx-popup-root")
     popupRoot = createRoot(container)
   }
 
+  if (!focusTrap) {
+    focusTrap = createFocusTrap(container, { clickOutsideDeactivates: true })
```

Comment:
Why do we both attach to the modal and create a focus trap? Would attaching to `aria-modal` not be sufficient for Trello to work again?

Reply:
probably, but i thought this would make it more robust.